lisa/sut_orchestrator/azure/common.py

```diff
@@ -543,6 +570,73 @@ def shared_gallery(self, value: Optional[SharedImageGallerySchema]) -> None:
         else:
             self.shared_gallery_raw = value.to_dict()  # type: ignore
 
+    @property
+    def community_gallery_image(self) -> Optional[CommunityGalleryImageSchema]:
```

Comment:
Please consider to reuse the logic, instead of copying them.

Reply:
updated